<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_pointcloud">
     <attribute name="title">
      <string>Pointcloud</string>
     </attribute>
     <widget class="QPushButton" name="bt_openplyfolder">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Open this folder</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_plyfoldername">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>351</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTreeWidget" name="plyfiles_treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>191</width>
        <height>391</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <column>
       <property name="text">
        <string>Filename</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="bt_calchistogram">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>380</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc Histogram</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_prefiximg">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>hist2_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_savehistimg">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>410</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save img</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_saveply">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save PLY</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_in_pointsize">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>000000000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pointsize">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Point size :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_voxelfilter">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0.01</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_voxelfilter">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Voxel filter points</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_voxelsize">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>75</y>
        <width>67</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Grid size</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_viewleft">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_viewright">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_camera">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Camera View </string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_viewdown">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_viewup">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_resetcam">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_x_rot">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_xrot">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>X-rot</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_y_rot">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_z_rot">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_yrot">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-rot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_zrot">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>240</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Z-rot</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_press_o">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>(press 'o' to swap pers/ortho)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pointcloud">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pointcloud</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_planeseg_1">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>280</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PlaneSeg1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_planeremove">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>310</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Plane removal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_planeseginlier">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>285</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>inlier distance</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_planeseg_inlier">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0.01</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_planecoef_a">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_planecoef_b">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_planecoef_c">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_planecoef_in">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>315</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>plan coef</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_planecoef_d">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_drawplan">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw Plan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_planestimate">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Plane Estimation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>105</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>meanK</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_outlier_meank">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>105</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>std dist</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_outlier_stddist">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_outlierremove">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Outlier remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_setpointorigin">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Set points origin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_planeseg_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>280</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PlaneSeg2</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_colormap">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentText">
       <string>Magma</string>
      </property>
      <item>
       <property name="text">
        <string>Magma</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inferno</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Plasma</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Viridis</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cividis</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Parula</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JetMod</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="bt_histgraph1">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>350</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>hist graph1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_histgraph2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>380</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>hist graph2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_histnameimg">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_postfiximg">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>410</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>-01</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_label">
     <attribute name="title">
      <string>Labelling</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_test1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_test2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_test1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelect_ply_folder"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_img_folder"/>
   </widget>
   <widget class="QMenu" name="menuPointcloud">
    <property name="title">
     <string>Pointcloud</string>
    </property>
    <addaction name="actionSet_points_origin_whole_folder"/>
   </widget>
   <widget class="QMenu" name="menuHistogramImg">
    <property name="title">
     <string>HistogramImg</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLabelImg">
    <property name="title">
     <string>LabelImg</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuShortcut">
    <property name="title">
     <string>Shortcut</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPointcloud"/>
   <addaction name="menuHistogramImg"/>
   <addaction name="menuLabelImg"/>
   <addaction name="menuShortcut"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSelect_ply_folder">
   <property name="text">
    <string>Select ply folder</string>
   </property>
  </action>
  <action name="actionSelect_img_folder">
   <property name="text">
    <string>Select img folder</string>
   </property>
  </action>
  <action name="actionSet_points_origin_whole_folder">
   <property name="text">
    <string>Set points origin whole folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
